package/package.json

```diff
@@ -20,6 +20,5 @@
     ],
     "website": "https://docs.mesosphere.com/current/usage/service-guides/spark/",
     "selected": true,
-    "version": "$version",
-    "minDcosReleaseVersion": "1.7"
+    "version": "{{package-version}}"
```

Comment:
Is there some error if we try to build a v2 universe without these changes?  I'd prefer not to have a pre-release translation step if it can be avoided.